ESP8266ControllerCircuit/ESP8266ControllerCircuit_cpl_jlc.csv

```diff
@@ -5,5 +5,5 @@ R2,4K7,R_0805_HandSoldering,19.500000,19.062500,180.000000,top
 R3,220R,R_0805_HandSoldering,19.500000,15.625000,180.000000,top
 R4,4K7,R_0805_HandSoldering,19.500000,12.187500,180.000000,top
 R5,4K7,R_0805_HandSoldering,19.500000,8.750000,180.000000,top
-U2,EL3H7(B)(TA)-G,SOP-4_4.4x2.8mm_Pitch1.27mm,57.250000,4.500000,180.000000,top
```

Comment:
Just tested it, and it needs to be 270 degrees
U2,EL3H7(B)(TA)-G,SOP-4_4.4x2.8mm_Pitch1.27mm,57.250000,4.500000,270.000000,top